package.json

```diff
@@ -33,7 +33,7 @@
   "homepage": "https://github.com/supabase/ssr#readme",
   "devDependencies": {
     "@eslint/js": "^9.3.0",
-    "@supabase/supabase-js": "^2.43.4",
+    "@supabase/supabase-js": "^2.56.0",
```

Comment:
Needs to be bumped further once this is live: https://github.com/supabase/supabase-js/pull/1545